### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by exploer** [musyoku/chainer-sru#2]
Hi， I have used your code for  other speech recognition task. and  I found that the speed is faster but it have bad result compared to result which use GRU. can you give me some advice? Thanks.

**Comment by musyoku** [comment]
How bad was the result?

**Comment by exploer** [comment]
CER：
   SRU: 30% 
   GRU: 18%

**Comment by exploer** [comment]
can you tell me which reason i should consider?

**Comment by musyoku** [comment]
you should change network architecture and hyperparameter(dropout, learning rate, optimizer, number of hidden layers, hidden layer dimention, etc)

**Comment by musyoku** [comment]
my implementation does not support variational dropout.
I don't know if variational dropout will help.

**Comment by exploer** [comment]
I used your cuda_optim code that implemented with pytorch. and I tried to change hyperparameter,but it seem does not work well.
